Classes/Hooks/Interfaces/TimeSpan.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+declare(strict_types=1);
+
+namespace OliverKlee\Seminars\Hooks\Interfaces;
+
+/**
+ * Use this interface for hooks concerning the date span.
+ *
+ * @author Michael Kramer <m.kramer@mxp.de>
+ */
+interface TimeSpan extends Hook
+{
+    /**
+     * Modifies the date span string.
+     *
+     * The date format for the date parts are configured in TypoScript (`dateFormatYMD` etc).
+     * This allows modifying the assembly of start and end date to the date span.
+     * E.g. for Hungarian: '01.-03.01.2019' -> '2019.01.01.-03.'.
```

Comment:
This should not be left to be fixed in a hook. Instead, I'd prefer we fix the original code instead and move the way shortened dates are assembled to the locallang files.